store/storetest/mocks/UserStore.go

```diff
@@ -848,3 +848,35 @@ func (_m *UserStore) VerifyEmail(userId string) store.StoreChannel {
 
 	return r0
 }
+
+// GetUserInteractions provides a mock function with given fields: uid, teamId, learningGroupPrefixes
+func (_m *UserStore) GetUserInteractions(uid string, teamId string, learningGroupPrefixes []*model.LearningGroupPrefix) store.StoreChannel {
+	ret := _m.Called()
+
+	var r0 store.StoreChannel
+	if rf, ok := ret.Get(0).(func() store.StoreChannel); ok {
+		r0 = rf()
+	} else {
+		if ret.Get(0) != nil {
+			r0 = ret.Get(0).(store.StoreChannel)
+		}
+	}
+
+	return r0
+}
+
+// GetUserTeams provides a mock function with given fields: uid, teamId, learningGroupPrefixes
+func (_m *UserStore) GetUserLearningGroups(uid string, teamId string, learningGroupPrefixes []*model.LearningGroupPrefix) store.StoreChannel {
+	ret := _m.Called()
```

Comment:
This looks like it is the same function as above with a different name. Consider either making a function factory for these, or by making an alias for the function?